### PR TITLE
Create separate entity list for Google

### PR DESCRIPTION
**Pull request by skim1102**
# About this PR
[Including all the regional Google domains](ttps://github.com/mozilla-services/shavar-prod-lists/pull/78) broke Shavar because it increased the entity list from ~300KB to 1.5MB. When creating the `mozstd-trackwhite-digest256`, remove the Google properties and resources and create a separate entity list google-trackwhite-digest256.

# Acceptance Criteria
- [ ] Version the entity lists so we can test increased file size limit and separate Google entity list only in Nightly (73)
- [ ] Version lower than the current Nightly (73) should continue to generate only mozstd-trackwhite-digest256 as the entity list. The entity list should still include Google domains, but not include regional changes in shavar-prod-lists as this will exceed the max size of the mozstd-trackwhite-digest256
- [ ] Version higher than or equal to the current Nightly should generate mozstd-trackwhite-digest256 without the Google domains and create additional list for just Google domains

#
~Dependent on issue https://github.com/mozilla-services/shavar-list-creation/issues/104 being completed.~ Removed from blocking this PR as it needs complete refactoring and we do not have to worry about reaching the limit _yet_.
Dependent on https://github.com/mozilla-services/shavar/pull/122.
Dependent on https://github.com/mozilla-services/shavar-list-creation-config/pull/58

### CODE REVIEW

File: publish2cloud.py [hunk 1]
```diff
@@ -259,6 +262,15 @@ def publish_to_cloud(config, chunknum, check_versioning=None):
             )
             if not versioning_needed:
                 continue
+
+            version = config.get(section, 'version')
+            ver = p_version.parse(version)
```
Comment (groovecoder) [inline publish2cloud.py]: 1-line this?
```python
version = p_version.parse(config.get(section, 'version'))
```

File: constants.py [hunk 2]
```diff
@@ -102,3 +107,9 @@
 LIST_TYPE_ENTITY = 'Entity'
 LIST_TYPE_PLUGIN = 'Plugin'
 LIST_TYPE_TRACKER = 'Tracker'
+
+SILLICON_VALLEY_COMPANIES = [
```
[the remark this answers is not in the document]
Comment (englehardt) [reply constants.py]: I recommend `LARGE_ENTITIES`.

File: sample_shavar_list_creation.ini [hunk 3]
```diff
@@ -127,6 +127,13 @@ output=mozplugin2-block-digest256
 [entity-whitelist]
 entity_url=https://s3.amazonaws.com/lists.disconnect.me/exampleservices.json
```
Comment (groovecoder) [inline sample_shavar_list_creation.ini]: This should be https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/disconnect-entitylist.json

File: constants.py [hunk 4]
```diff
@@ -102,3 +107,9 @@
...
+    'Google',
+]
+
+VER_SV_SEPARATION_STARTED = 73
```
[the remark this answers is not in the document]
Comment (skim1102) [reply constants.py]: Done!

**Comment by skim1102** [comment]
Please do not merge until Staging Shavar is ready.

**Comment by skim1102** [comment]
Since we are trying to introduce a new list to the pref on the client side before the pref change is published, we need Google entities still in the standard entity list along with `google-trackwhite-digest256` list being available to be served by Shavar.
Commits a995574  adbce7a enables that. To test:
- [ ] standard entity list generated in master/nightly branch and from versioned branch is the same size
- [ ] `google-trackwhite-digest256` is created on the master/nightly folder/bucket/path in S3
Issue created to revert this in the future https://github.com/mozilla-services/shavar-list-creation/issues/114

**Comment by groovecoder** [comment]
Spot check also worked!

**Pull request merged**